nvm.sh

```diff
@@ -2911,10 +2911,18 @@ nvm() {
           -s)
             shift # consume "-s"
             nobinary=1
+            if ["${nosource}" -eq 1]; then
```

Comment:
```suggestion
            if [ $nosource -eq 1 ]; then
```
quotes aren't needed for -eq